Source/WebKit/Shared/Cocoa/CoreIPCPassKit.mm

```diff
@@ -33,26 +33,33 @@
 namespace WebKit {
 
 CoreIPCPKContact::CoreIPCPKContact(PKContact *contact)
-    : m_name(contact.name)
-    , m_emailAddress(contact.emailAddress)
-    , m_phoneNumber(contact.phoneNumber)
-    , m_postalAddress(contact.postalAddress)
+    : m_emailAddress(contact.emailAddress)
 ALLOW_DEPRECATED_DECLARATIONS_BEGIN
     , m_supplementarySublocality(contact.supplementarySubLocality)
 ALLOW_DEPRECATED_DECLARATIONS_END
 {
+    if (contact.name)
```

Comment:
These could be still in the initializers with ternary operators

Reply:
That was the first attempt, but ternary + std::optional == a bad time.